packages/widgets-manager/src/__tests__/grid.spec.ts

```diff
@@ -0,0 +1,184 @@
+import { GridManager } from '../grid';
+import { AppendAppStrategy } from '../strategies/mounting/append';
+import { App } from '../types';
+const windowObject: Window = window;
+
+const mockApp1: App = {
+  id: 1,
+  name: 'Test App',
+  payload_type: 'html',
+  payload: '',
+  settings: {
+    css: '',
+    inlineCss: {},
+    position: {} as any,
+  },
+  organization_id: 1,
+}
+
+const mockApp2: App = {
+  id: 2,
+  name: 'Test App2',
+  payload_type: 'html',
+  payload: '',
+  settings: {
+    css: '',
+    inlineCss: {},
+    position: {} as any,
+  },
+  organization_id: 1,
+}
+
+describe('GridManager tests', () => {
+  const settings = {
+    columns: 3,
+    gutter: 10,
+    padding: 10,
+    positions: [
+      'top-left',
+      'top-center',
+      'top-right',
+      'center-left',
+      'center-center',
+      'center-right',
+      'bottom-left',
+      'bottom-center',
+      'bottom-right',
+    ]
+  };
+  const addStrategy = new AppendAppStrategy();
+
+  it('I see GridManager correctly created and configured', () => {
+    const gm = new GridManager(settings, windowObject, addStrategy);
+    expect(gm.grid.cells.length).toBe(9);
+    expect(gm.grid.cells).toContainEqual(
+      {
+        id: 'top-left',
+        apps: [],
+        position: {
+          x: 0,
+          y: 0,
+        },
+        size: {
+          width: windowObject.innerWidth / 3,
+          height: windowObject.innerHeight / 3,
+        }
+      }
+    );
+
+    expect(gm.grid.cells).toContainEqual(
+      {
+        id: 'top-right',
+        apps: [],
+        position: {
+          x: 2 * (windowObject.innerWidth / 3),
+          y: 0,
+        },
+        size: {
+          width: windowObject.innerWidth / 3,
+          height: windowObject.innerHeight / 3,
+        }
+      }
+    );
+
+  });
+
+  it('GridManager _setGridDimensions works on different scenarios', () => {
```

Comment:
Here, you are **describing** `_setGridDimensions`.
And "works on different scenarios" doesn't really mean anything